Ingestion/templates/datahub-executor.ecs.template.yaml

```diff
@@ -0,0 +1,668 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::LanguageExtensions
+Metadata:
+    AWS::CloudFormation::Interface:
+      ParameterGroups:
+        - Label:
+            default: Network
+          Parameters:
+            - VPCID
+            - SubnetID
+        - Label:
+            default: AWS Environment
+          Parameters:
+            - AwsRegion
+        - Label:
+            default: DataHub Info
+          Parameters:
+            - DataHubBaseUrl
+            - ExecutorId
+            - ExistingDataHubAccessTokenSecretArn
+            - DataHubAccessToken
+            - DatahubIngestionsMaxWorkers
+            - DatahubMonitorsMaxWorkers
+            - DatahubIngestionsSignalPollInterval
+            - OptionalSecrets
+            - OptionalEnvVars
+        - Label:
+            default: Ingestion Container Task
```

Comment:
Let's rename this...

Worker Tasks

Reply:
- Network Parameters
- DataHub Parameters
- Worker Task Parameters